### PR TITLE
Adding geolocation and openweathermap

### PR DESCRIPTION
**Pull request by JeesubKim**
Geolocation feature has been updated so that center of map will be set as user's location.

If user set openweathermap API as props with geolocation, 
Weather information of location will be displayed with pop up.

### CODE REVIEW

File: src/MapBox.js [hunk 1]
```diff
@@ -117,21 +127,108 @@ function MapBox(props) {
         mapOption.maxZoom = props.slideshow.maxZoom
         mapOption.minZoom = props.slideshow.minZoom 
       }
-      
+      console.log(mapOption);
```
Comment (soaple) [inline src/MapBox.js]: Please remove console.log().

File: package.json [hunk 2]
```diff
@@ -3,13 +3,15 @@
     "version": "0.1.0",
     "description": "MapBox component for StickyBoard",
     "main": "dist/index.js",
+
```
Comment (soaple) [inline package.json]: Please remove this empty line.

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the comments.

**Comment by JeesubKim** [comment]
Please check update

**Review by soaple — APPROVED** [review]
LGTM

**Pull request merged**